tests/test_detect_target.py

```diff
@@ -47,98 +69,91 @@ def image_zidane():
     """
     Load Zidane image.
     """
-    image = cv2.imread(IMAGE_ZIDANE_PATH)
+    image = cv2.imread(str(IMAGE_ZIDANE_PATH))
     result, zidane_image = image_and_time.ImageAndTime.create(image)
     assert result
     assert zidane_image is not None
     yield zidane_image
 
+@pytest.fixture()
+def expected_bus():
+    """
+    Load expected bus bounding box.
+    """
+    expected_bus = np.loadtxt(BOUNDING_BOX_BUS_PATH)
```

Comment:
Add a comment explaining the format of the text file (e.g. `confidence label bounds` ).